conda_smithy/lint_recipe.py

```diff
@@ -660,7 +660,9 @@ def lintify(meta, recipe_dir=None, conda_forge=False):
         expected_exceptions = set([l.strip() for l in expected_exceptions])
     if set(filtered_licenses) - expected_licenses:
         hints.append(
```

Comment:
To require spdx we would need to make this a lint.

Reply:
Sorry I'm not trying to change hint/lint. Just trying to add the link. I'll update the note accordingly.